README.md

```diff
@@ -1,2 +1,11 @@
 # password-generator
-Built a password generator focusing on Javascript
+
+Built a password generator focusing on Javascript.
+
+## Link to webpage.
+
+[link:] (https://jj77847.github.io/password-generator/)
+
+### Screenshot of website
+
+<img width="1440" alt="Website Screenshot" src="./Assets/images/password-generator-screenshot.png">
```

Comment:
This screenshot isn't working. I think it may be a path issue so please address this.

Reply:
retook screen shot and added correct links